thenewboston_node/blockchain/tests/test_get_blockchain_state_meta.py

```diff
@@ -33,7 +33,7 @@ def test_can_get_blockchain_genesis_state_meta(api_client, file_blockchain, bloc
 
     assert response_json == {
         'last_block_number':
-            blockchain_genesis_state.last_block_number,
+            blockchain_genesis_state.message.last_block_number,
```

Comment:
let's have `blockchain_genesis_state.get_last_block_number()` and use it everywhere